dm/ctl/master/operate_source.go

```diff
@@ -31,7 +31,7 @@ import (
 // NewOperateSourceCmd creates a OperateSource command
 func NewOperateSourceCmd() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "operate-source <operate-type> [config-file ...] [--print-sample-config]",
+		Use:   "operate-source <operate-type> [config-file | sourceID ...] [--print-sample-config]",
```

Comment:
How about updating the message of `operate-source create/update/stop should specify config-file(s)`?

Reply:
what's the correct way to express 😂 
```
operate-source create|update [config-file ...] [--print-sample-config]
operate-source stop [config-file | sourceID ...] [--print-sample-config]
operate-source show [--print-sample-config]
```
is this OK?

Reply:
OK. 😂

Reply:
https://github.com/pingcap/dm/pull/855/commits/3780ae509e1da536483d49d6f6dd9f6c49801cc5

Reply:
multi-line usage is quite hard using cobra

```
[2020-08-04T04:16:35.427Z] Available Commands:
[2020-08-04T04:16:35.427Z]   check-task          	check-task <config-file>
[2020-08-04T04:16:35.427Z]   get-task-config     	get-task-config <task-name> [--file filename]
[2020-08-04T04:16:35.427Z]   list-member         	list-member [--leader] [--master] [--worker] [--name master-name/worker-name ...]
[2020-08-04T04:16:35.427Z]   migrate-relay       	migrate-relay <source> <binlogName> <binlogPos>
[2020-08-04T04:16:35.427Z]   offline-member      	offline-member <--master/--worker> <--name master-name/worker-name>
[2020-08-04T04:16:35.427Z]   operate-leader      	operate-leader <operate-type>
[2020-08-04T04:16:35.427Z]   operate-schema      	operate-schema <operate-type> <-s source ...> <task-name> <-d database> <-t table> [schema-file]
[2020-08-04T04:16:35.427Z]   operate-source      	operate-source create|update [config-file ...] [--print-sample-config] [flags]
[2020-08-04T04:16:35.427Z]         operate-source stop [config-file | sourceID ...] [--print-sample-config] [flags]
[2020-08-04T04:16:35.427Z]         operate-source show [--print-sample-config] [flags]
```

Reply:
how about just hide support `sourceID`, and find a better way to display when we refactor subcommand of https://github.com/pingcap/dm/issues/398

Reply:
> how about just hide support sourceID, and find a better way to display when we refactor subcommand of #398

OK

Reply:
revert in https://github.com/pingcap/dm/pull/855/commits/b41bf2c225e91b2bb5f10ef8d34dce0bc4950b8e